lib/azure/armrest/resource_group_based_service.rb

```diff
@@ -46,7 +46,16 @@ def list(rgroup = configuration.resource_group)
         url = yield(url) || url if block_given?
         response = rest_get(url)
 
-        Azure::Armrest::ArmrestCollection.create_from_response(response, model_class)
+        results = Azure::Armrest::ArmrestCollection.create_from_response(response, model_class)
+
+        while results.continuation_url
+          response = rest_get_without_encoding(results.continuation_url)
+          more = Azure::Armrest::ArmrestCollection.create_from_response(response, model_class)
+          results.concat(more)
+          results.continuation_url = more.continuation_url
+        end
+
+        results
       end
```

Comment:
@bronaghs Maybe it's not even worth exposing this as an accessor to begin with. I'll update the PR to remove it and just handle it internally for now since it's only used in 2 methods at the moment. We could always revisit.